app/lib/results.rb

```diff
@@ -4,8 +4,18 @@ def initialize(left, right)
     @result_count = right['results'].count > left['results'].count ? right['results'].count : left['results'].count
     @left = (0..result_count - 1).map { |i| Result.new(left['results'][i]) }
     @right = (0..result_count - 1).map { |i| Result.new(right['results'][i]) }
-    @left_total = left['total']
-    @right_total = right['total']
+
+    @left_total = if left['total'].present?
+                    left['total']
+                  else
+                    left['result_count']
+                  end
```

Comment:
Could use a method which can cater for both left and right? Something like:

```
def totals(result)
  result['total'].present? ? result['total'] : result['result_count']
end
```

`@left_total   = totals(left)`
`@right_total = totals(right)`

You can think of a better name for the method and parameter :)

Reply:
Again here the `present?` is not required

instead try:

```
result['total'] || result['result_count']
```
however before does this I would like to know why we have two different keys for this? as the data should be in a consistent view.

Reply:
for some reason it's not, no result['total'] on staging :/ did wonder if i was doing something wrong